packages/ui-components/src/Common/AlertBox/index.module.css

```
@@ -15,6 +15,10 @@
     text-sm
     text-white;
 
+  p {
+    color: white !important;
```

```suggestion
    @apply text-white;
```

What if you do something like that ?

---

ok